Author: Daigo Moriwaki <daigo@debian.org>
Description: Exclude files from the RDoc generation.
- mkconfig.rb, which causes an error.
- test_*.rb, which are test cases. 
--- a/common.mk
+++ b/common.mk
@@ -261,7 +261,7 @@ post-install-doc::
 
 rdoc: $(PROGRAM) PHONY
 	@echo Generating RDoc documentation
-	$(RUNRUBY) "$(srcdir)/bin/rdoc" --all --ri --op "$(RDOCOUT)" "$(srcdir)"
+	$(RUNRUBY) "$(srcdir)/bin/rdoc" --all --ri --op "$(RDOCOUT)" --exclude mkconfig.rb --exclude test_ "$(srcdir)"
 
 what-where-doc: no-install-doc
 no-install-doc: pre-no-install-doc dont-install-doc post-no-install-doc
